internal/renderer/blockquote.go

```diff
@@ -0,0 +1,49 @@
+// This file is part of gmnhg.
+
+// gmnhg is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// gmnhg is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with gmnhg. If not, see <https://www.gnu.org/licenses/>.
+
+// Package renderer contains an implementation of markdown => text/gemini
```

Comment:
The godoc package comment is currently copied into every file. For the split file package approach, it's usually done with a file like `doc.go` containing just `package packagename` and the godoc comment.

Reply:
Thanks, I wasn't aware of that. Fixed in latest push